secretmanager/README.md

```diff
@@ -0,0 +1,59 @@
+# Create and Access Secrets in Secret Manager
+
+In this example you'll see how to use the [Workflow Connector to Secret Manger](https://cloud.google.com/workflows/docs/reference/googleapis/secretmanager/Overview) to **create** and **access** `string` secrets in Secret Manager.
+
+More specifically, you will use Secret Manager's [create](https://cloud.google.com/workflows/docs/reference/googleapis/secretmanager/v1/projects.secrets/create) and [addVersionString](https://cloud.google.com/workflows/docs/reference/googleapis/secretmanager/v1/projects.secrets/addVersionString) methods to create a secret, and [accessString](https://cloud.google.com/workflows/docs/reference/googleapis/secretmanager/v1/projects.secrets.versions/accessString) method to read the value of a secret.
+
+## Create a Secret
+
+First, you need to specify the GCP project where you want to store the secret. You can use the `GOOGLE_CLOUD_PROJECT_ID` environment variable to get the current project ID.
+
+```yaml
+- Get Project ID:
```

Comment:
This can be much simpler like:

- projectID: ${sys.get_env("GOOGLE_CLOUD_PROJECT_ID")}

Reply:
done